apps/datahub/src/app/record/record-metadata/record-metadata.component.html

```diff
@@ -23,16 +23,7 @@
         >
         </gn-ui-metadata-info>
       </div>
-      <div>
-        <div *ngIf="metadataQualityDisplay">
-          <p class="text text-gray-700 text-xs mb-3 uppercase" translate>
-            record.metadata.quality
-          </p>
-          <gn-ui-metadata-quality
-            [metadata]="facade.metadata$ | async"
-            [metadataQualityDisplay]="metadataQualityDisplay"
-          ></gn-ui-metadata-quality>
-        </div>
```

Comment:
hmm, was there a reason to remove that? This indicator should stay somewhere on the page, although I agree that now, above the thumbnail is probably not the best.

Also our E2E tests are lacking because we're not testing that this indicator indeed show up :grimacing:

Reply:
Yes ok, I see what you mean. These kind of "hidden" components are tricky. This component is important for platforms which use it, but enabling it is not necessarily straightforward so we tend to forget them. See 

@lucilecollignon do you have a suggestion on what do to with this component? Currently it looks like this:
![image](https://github.com/geonetwork/geonetwork-ui/assets/10629150/5527f5ed-1695-4c00-9b7f-fedfd13652ab)

Reply:
Interesting, I never saw this component before :slightly_frowning_face: 

Waiting for Lucile's input, but I feel like it should go under the lightbox and contact information.